docs/integrations/destinations/duckdb.md

```diff
@@ -1,16 +1,28 @@
 # DuckDB
 
-:::danger
+:::caution
```

Comment:
You can make the documentation cloud/oss specific by using these conditional comments: https://github.com/airbytehq/airbyte/blob/646ccdf22dcfe8dabb753991724f32b56594dd76/docs/integrations/destinations/s3.md#L60

Reply:
Nice 👍

Reply:
Done 👍